modules/search/instant-search/components/search-app.jsx

```diff
@@ -69,9 +69,9 @@ class SearchApp extends Component {
 	prepareDomForMounting() {
 		// Clean up the page prior to mounting component
 		hideElements( this.props.themeOptions.elementSelectors );
-		document
-			.querySelectorAll( '.jetpack-instant-search-wrapper' )
-			.forEach( widget => removeChildren( widget ) );
+		//		document
```

Comment:
The Instant Search code was removing the Jetpack search widget I added to my site. Rather than figuring out the right way to test instant search, I just commented this code out :)

This change should be reverted before shipping.